SocialFilter/sections/css_extended.txt

```diff
@@ -102,6 +102,7 @@ journals.lww.com#?#.ejp-article-tools__list > .ejp-article-tools__list-item:has(
 enoteca.co.jp#?#.justify-end > .md\:text-left:contains(/^この記事をシェア$/)
 ichijin-plus.com#?#main > div[class^="sc-"][class*="sc-"] > div[class^="sc-"]:has(> div[class] a[href^="https://twitter.com/intent/tweet?"])
 creators.yahoo.co.jp#?#.ArticleDetail__actionTitle:contains(シェア)
+finance.yahoo.co.jp#?#div[class^="_"]:has(> ul > li > a[href="#"][target="_blank"])
```

Comment:
@SKEIDs please check this on your end
```
finance.yahoo.co.jp#?#div[class^="_"]:has(> ul > li > a[href="#"][target="_blank"][aria-label^="Facebook"])
```

Reply:
But I don't like `#?#div[class^="_"]:has`, too many nodes to evalutate.